frontend/src/metabase-types/api/mocks/metadata.ts

```diff
@@ -0,0 +1,16 @@
+import Metadata from "metabase-lib/metadata/Metadata";
+import { createMockDatabase } from "./database";
+import { createMockTable } from "./table";
+
+export function createMockMetadata(options: Partial<Metadata> = {}) {
+  return new Metadata({
+    databases: { 1: createMockDatabase() },
+    tables: { 1: createMockTable() },
+    questions: {},
+    schemas: {},
+    fields: {},
+    metrics: {},
+    segments: {},
+    ...options,
+  });
+}
```

Comment:
`metabase-types` shouldn't import stuff from `metabase-lib`. Can we move this to `metabase-lib/mocks` for now maybe? Or maybe wait until #28836 is merged and use it instead

Reply:
FWIW #28836 is merged and IMO we should try to use it here instead